packages/paste-website/src/pages/components/help-text/index.mdx

```diff
@@ -0,0 +1,375 @@
+    <Option value="messaging">SMS</Option>
+    <Option value="numbers">Phone Numbers</Option>
+    <Option value="video">Video</Option>
+  </Select>
+  <HelpText variant="default">Please choose a Twilio product.</HelpText>
+</>`}
+</LivePreview>
+
+### Help text with textarea
+
+<LivePreview scope={{Label, HelpText, TextArea}} language="jsx">
+  {`<>
+  <Label htmlFor="message" required>Message (at least 120 characters)</Label>
+  <TextArea onChange={()=>{}} onChange={()=>{}} aria-describedby="message_help_text" id="message" name="message" placeholder="Enter message" required />
+  <HelpText id="message_help_text">Please enter at least 120 characters</HelpText>
+</>`}
+</LivePreview>
+
+## States
+
+### Default help text
+
+Give users enough information in help text to prevent input and formatting errors. Keep it concise and scoped to information that will help with validation. For example, use help text if a password field has specific requirements that a user should know prior to filling it out.
+
+- Help text should have enough information to help users prevent errors.
+- Use help text to provide instruction if needed. For example, don't use "Enter the date you wish to receive your bill below" as label text. Instead, use "Billing date" as a label and "Your account will be automatically billed on the above date." as help text.
+
+<LivePreview scope={{Input, Label, HelpText}} language="jsx">
+  {`<>
+  <Label htmlFor="email_default_example">Email address</Label>
+  <Input aria-describedby="email_default_help_text" id="email_default_example" name="email_default_example" type="email" placeholder="example@twilio.com" onChange="" />
+  <HelpText id="email_default_help_text" variant="default">Please enter a valid email.</HelpText>
+</>`}
+</LivePreview>
+
+### Error help text
+
+An inline error is placed below the field to inform a user of any errors in their value. If help text exists, the error text should replace and repeat the help text.
+If only rendering error text, be sure to include the appropriate attributes on the form element and the help text.
+
+Error text should:
+
+- Be actionable. Explain how to fix an error and if reasonable, why it happened so that it might also be prevented in the future.
+- Be concise and simple, maybe even more than normal. Avoid jargon. Try to keep error text to 2 sentences or fewer.
+- Use the passive voice for input errors to avoid placing blame on the user. For example, "A friendly name is required."
+- Use the active voice for system errors. For example, "Our systems are currently down. Please contact our support team."
+
+<LivePreview scope={{Input, Label, HelpText}} language="jsx">
+  {`<>
+  <Label htmlFor="email_error_example">Email address</Label>
+  <Input aria-describedby="email_error_help_text" id="email_error_example" value="this is not a vaild entry" name="email_error_example" type="email" placeholder="example@twilio.com" onChange="" hasError />
```

Comment:
Using `defaultValue` instead of `value` might be a little less confusing for people who don't understand why they can't interact with the Input